docs/changelog.md

```diff
@@ -12,6 +12,12 @@ See the [Contributing Guide](contributing.md) for details.
 
 ## [unreleased]
 
+### Fixed
+
+* Fix `codecs` deprecation.
```

Comment:
Let's note the Python version where the codec deprecation is happening for future reference.